com.woltlab.wcf/clipboardAction.xml

```diff
@@ -180,5 +180,40 @@
 				<page>wcf\acp\page\ArticleListPage</page>
 			</pages>
 		</action>
+		<action name="assignUserByClipboard">
```

Comment:
Why this elaborate name?

Reply:
`assignUser` is already used in `ModerationQueueAction` so that we need to use a different action name here which is mapped to the DBO action name directly.